### PR TITLE
Fix workflow failures

### PR DESCRIPTION
**Pull request by karthikmsqa**
## Overview

Fix cucumber test workflow failures for validation changes that recently went in

### CODE REVIEW

File: .github/workflows/cucumber-tests.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Cucumber Tests
 
 on:
   push:
-    branches: [master]
+    branches: [master, qa-test-new-code-changes]
```
Comment (sdevalapurkar-bcgov) [inline .github/workflows/cucumber-tests.yml:5]:
```suggestion
    branches: [master]
```

**Review by sdevalapurkar-bcgov — APPROVED** [review]
nice fix

**Review by sdevalapurkar-bcgov — DISMISSED** [review]
LGTM

File: tests/src/test/java/stepDefinitions/backendstepdefinitions/CreateCsoAccountTest.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import javax.xml.bind.SchemaOutputResolver;
```
Comment (sdevalapurkar-bcgov) [inline tests/src/test/java/stepDefinitions/backendstepdefinitions/CreateCsoAccountTest.java:16]: can this be removed?
```suggestion
```

**Review by sdevalapurkar-bcgov — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -7,4 +7,4 @@ tests/logs
 .idea
 tests/test-output
 .env
-src/test/resources/local.properties
+tests/src/test/resources/local.properties
```
Comment (sdevalapurkar-bcgov) [inline .gitignore:10]: how come this was changed? was the file location changed?

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontendutils/DriverClass.java [hunk 4]
```diff
@@ -42,9 +42,9 @@ public void initializeDriver() {
                     log.info("URL value is not available in the properties file.");
             }
         }
-        public void  driverSetUp() throws IOException {
+        public void  driverSetUp() {
```
Comment (sdevalapurkar-bcgov) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontendutils/DriverClass.java:45]: how come `throws IOException` was removed here and also below?

File: .gitignore (hunk 3 above)
Comment (karthikmsqa) [reply .gitignore:10]: Location is not changed. File wasn't ignored in the earlier commit. Checking with path with content root instead of from source.

Comment (sdevalapurkar-bcgov) [reply .gitignore:10]: ok so it was wrong earlier and this is the right path

Comment (karthikmsqa) [reply .gitignore:10]: That's correct.

**Review by sdevalapurkar-bcgov — APPROVED** [review]

**Pull request merged**